Modules/HMPID/include/HMPID/HmpidTaskMatches.h

```diff
@@ -0,0 +1,75 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
+// // distributed under the terms of the GNU General Public License v3 (GPL
+// // Version 3), copied verbatim in the file "COPYING".
+// //
+// // See http://alice-o2.web.cern.ch/license for full licensing information.
+// //
+// // In applying this license CERN does not waive the privileges and immunities
+// // granted to it by virtue of its status as an Intergovernmental Organization
+// // or submit itself to any jurisdiction.
+//
+
+///
+/// \file   HmpidTaskMatches.h
+/// \author Nicola Nicassio, Giacomo Volpe
+///
+
+#ifndef QC_MODULE_HMPID_HMPIDTASKMATCHES_H
+#define QC_MODULE_HMPID_HMPIDTASKMATCHES_H
+
+#include <Framework/InputRecord.h>
+#include "QualityControl/TaskInterface.h"
+
+#include "DataFormatsGlobalTracking/RecoContainer.h"
+#include "ReconstructionDataFormats/MatchInfoHMP.h"
+#include "SimulationDataFormat/MCCompLabel.h"
+
+#include <TH1.h>
```

Comment:
a forward declaration of these classes should be enough instead of an include

Reply:
ok!